docs/docs/training-data-format.mdx

```diff
@@ -380,26 +380,26 @@ stories:
 
 Each step can be one of the following:
 
-  - A [user message](#user-messages), represented by **intent** and **entities**.
-  - An [or statement](#or-statement), which includes two or more user messages under it
-  - A bot [action](#actions)
-  - A [form](#forms)
-  - A [slot was set](#slots) event
-  - A [checkpoint](#checkpoints), which connects the story to another story
```

Comment:
@akelad @amn41 for my understanding: do you propose to return that to original state as well? So we present data format as if e2e doesn't exist?

Reply:
yeah i don't think we should have the end to end format on an overview level if we still sell it as an experimental feature

Reply:
ok, thank you for the clarification. Reworking my changes

Reply:
yes I think that's what we should do